src/Events/Substrate/Marketplace/AuctionFinalized.php

```diff
@@ -6,43 +6,27 @@
 use Enjin\Platform\Events\PlatformBroadcastEvent;
 use Illuminate\Broadcasting\Channel;
 use Illuminate\Database\Eloquent\Model;
+use Enjin\Platform\Services\Processor\Substrate\Codec\Polkadart\Events\Marketplace\AuctionFinalized as AuctionFinalizedPolkadart;
 
 class AuctionFinalized extends PlatformBroadcastEvent
 {
     /**
      * Create a new event instance.
      */
-    public function __construct(Model $listing, Model $state, Model $sale, ?Model $transaction = null)
+    public function __construct(AuctionFinalizedPolkadart $event, ?Model $transaction = null, ?array $extra = null)
     {
         parent::__construct();
 
-        $this->model = $sale;
```

Comment:
Did we remove this for platform-core as well? This is attribute being used in https://github.com/enjin/platform-multi-tenant/blob/master/src/Listeners/LinkEventDataToTenant.php#L24

Reply:
Here as well https://github.com/enjin/platform-multi-tenant/blob/master/src/Listeners/LinkRelayTransactionToTenant.php#L18

Reply:
Yep, it was removed from there. I also asked here: https://enjin.slack.com/archives/C04LN0G5F2P/p1717166162432909
I can check what I can do to put it back, no problem.